server/fleet/apple_mdm.go

```diff
@@ -25,6 +25,34 @@ const (
 	MDMAppleEnrollmentTypeManual MDMAppleEnrollmentType = "manual"
 )
 
+type MDMAppleDeliveryStatus string
+
+var (
+	MDMAppleDeliveryFailed  MDMAppleDeliveryStatus = "failed"
+	MDMAppleDeliveryApplied MDMAppleDeliveryStatus = "applied"
+	MDMAppleDeliveryPending MDMAppleDeliveryStatus = "pending"
+)
+
+func MDMAppleDeliveryStatusFromCommandStatus(cmdStatus string) *MDMAppleDeliveryStatus {
```

Comment:
Should we consider adding constants for the Apple command statuses too? Or maybe there are constants that nano is already using?

Reply:
I took a look: neither nano or micro are using just strings, but I think this is a good idea, so I defined a set of constants.